Mathlib/Topology/Separation/Regular.lean

```diff
@@ -637,8 +637,8 @@ theorem connectedComponent_eq_iInter_isClopen [T2Space X] [CompactSpace X] (x :
           ⟨s ∩ v, H2, mem_inter H.2.1 h1⟩
 
 /-- `ConnectedComponents X` is Hausdorff when `X` is Hausdorff and compact -/
+@[stacks 0900 "The Stacks entry proves profiniteness."]
```

Comment:
I can't spot my syntax error here

Reply:
Removing the string gives `unknown attribute [stacks]`, so just a missing import (pushed a fix)

Reply:
Ah 🤦